yb-voyager/cmd/exportData.go

```diff
@@ -80,6 +80,12 @@ func exportData() {
 	success := exportDataOffline()
 
 	if success {
+		tableRowCount := datafile.OpenDescriptor(exportDir).TableRowCount
+		printExportedRowCount(tableRowCount)
+		callhome.GetPayload(exportDir)
```

Comment:
Just add a comment here for clarification that it initialises the required in-memory payload structure along with the file/filepath if not already present- this should fix the issue 👍🏽 
All other calls to this function explicitly make use of the Payload struct it retrieves

Reply:
updated the description of `GetPayload()`function itself